1_iris_simple_example/requirements.txt

```diff
@@ -0,0 +1,5 @@
+cleanlab
```

Comment:
let's remove cleanlab from all the per-example requirements.txt files.
In the future for examples that will only work with master branch of cleanlab, we can put a comment on the top of this file that say so, with commented out pointer to execute: `pip install git+https://github.com/cleanlab/cleanlab.git`

Reply:
whether that's worthwhile for the new examples we just added is up to you (if you  add this disclaimer now, need to remember to remove it after we release 2.1 in a few days)